apps/web/src/components/docs/Docs.tsx

```diff
@@ -81,6 +97,10 @@ export const Docs = ({ code = '', description = '', title = '', isLoading, child
   }
 
   if (Component === null) {
+    if (isChildDocs) {
+      return <Text>We could not load this part of the documentation for you. Please try again.</Text>;
```

Comment:
In what scenario will this happen? And what will change if the user will try again?

Reply:
If the docs could not be fetched from github for some reason. It could if it is github that is the issue. For me it is more a being nice to the user thing.